enterprise/frontend/src/embedding-sdk/cli/steps/setup-license.ts

```diff
@@ -54,6 +57,8 @@ export const setupLicense: CliStepMethod = async state => {
         },
       });
 
+      spinner.start("Activating the license...");
```

Comment:
Activating the license can be a bit slow when the internet connection is slow (I assume contacting the Metabase license server), so we add a loading state.